tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -13,6 +13,20 @@ use super::parameters::{CiphertextConformanceParams, CiphertextListConformancePa
 #[derive(Debug, PartialEq, Eq, Copy, Clone, Serialize, Deserialize)]
 pub struct Degree(pub usize);
 
+/// This tracks the amount of noise in a ciphertext.
```

Comment:
docstring does not look right, the maximum amount of noise allowed in a [`Ciphertext`] maybe ?

Reply:
Fixed